posts/inside-rust/2020-07-09-lang-team-path-to-membership.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: "Lang team design meeting: path to membership"
+author: Niko Matsakis
+description: "Lang team design meeting: path to membership"
+team: the lang team <https://www.rust-lang.org/governance/teams/lang>
+---
+
+Today the [lang team] design meeting was on the topic of the "path to
+membership". This blog post gives a brief summary; you can also read
+the [minutes] or view the [recording].
+
+[minutes]: https://github.com/rust-lang/lang-team/blob/master/design-meeting-minutes/2020-07-08-lang-team-path-to-membership.md
+[lang team]: https://www.rust-lang.org/governance/teams/lang
+[recording]: https://youtu.be/SeH-hZgDG1Y
+
+The premise of the meeting was that the lang team has never had a
+particularly clear *path to membership* -- i.e., it's been hard to say
+exactly what are the kinds of steps that one should be taking if you
+would like to become a member of the lang team. However, with the
+shift to [major change proposals] and in particular [project groups],
+we're starting to see what such a path looks like. 
+
+[major change proposals]: https://github.com/rust-lang/rfcs/pull/2936/
+[project groups]: https://github.com/rust-lang/rfcs/pull/2856
+
+### Expectations for a team member
+
+As part of our discussion, we came up with a relatively complete list of
+what we see as the "expectations for a lang-team member". To be clear,
+this is the full set of possible expectations: many members only have the
+time to do some subset of these things at any given time.
+
+- Lead project groups, where appropriate
+- Liaison for projects, where appropriate
+- Participate in triage meetings
+- Participate in design meetings
+- Respond to rfcbot fcp requests in a timely fashion
+- Participate constructively in, and help facilitate, RFC discussion, issues, PRs, and other GitHub-based discussions
+    - Provide important technical points
+    - Help to drive discussions towards common understanding
+    - Understanding and documenting the positions and points being raised
+- Monitor and respond to communication in Zulip
+
+### A sketch for a path to membership
+
+The core idea for a path to membership is that we want some set of
+steps that let us see people doing the things we expect from lang-team
```

Comment:
Doing the things and demonstrating the qualities that we expect of lang team members